web/packages/plugins/analysis-platform/src/modules/Analysis/components/AnalysisJobTable.tsx

```diff
@@ -178,6 +179,7 @@ const JobRow = (props: {
 
 export const AnalysisJobTable = (props: AnalysisJobTableProps) => {
   const { jobs, analysisId, dataSourceId, setJobs } = props
+  // @ts-ignore
```

Comment:
the type error should be fixed in master branch, and not necessary to add the @ts-ignore